apis/project.cattle.io/v3/schema/schema.go

```diff
@@ -584,7 +584,6 @@ func addServiceOrDNSRecord(dns bool) types.SchemasInitFunc {
 		schemas = schemas.
 			AddMapperForType(&Version, v1.ServiceSpec{},
 				&m.Move{From: "externalName", To: "hostname"},
```

Comment:
shall we also make clusterIP field non-updatable? there is norman `noupdate` tag

Reply:
It is non-updatable already in https://github.com/rancher/types/blob/master/apis/project.cattle.io/v3/schema/schema.go#L625